examples/dreamcast/kgl/basic/elements/gl-elements.c

```diff
@@ -78,6 +78,22 @@ void RenderCallback(GLuint texID) {
     glDisable(GL_KOS_NEARZ_CLIPPING);
 }
 
+int check_start(void) {
```

Comment:
```suggestion
static int check_start(void) {
```
Making these `static`, cuz they are never used from outside of this source file/translation unit.